toplevel/coqtop.ml

```diff
@@ -30,25 +30,18 @@ let print_header () =
   Feedback.msg_notice (str "Welcome to Coq " ++ str ver ++ str " (" ++ str rev ++ str ")");
   flush_all ()
 
-let memory_stat = ref false
 let print_memory_stat () =
-  begin (* -m|--memory from the command-line *)
-    if !memory_stat then
-    Feedback.msg_notice
-      (str "total heap size = " ++ int (CObj.heap_size_kb ()) ++ str " kbytes" ++ fnl ());
-  end;
-  begin
-    (* operf-macro interface:
-       https://github.com/OCamlPro/operf-macro *)
-    try
-      let fn = Sys.getenv "OCAML_GC_STATS" in
-      let oc = open_out fn in
-      Gc.print_stat oc;
-      close_out oc
-    with _ -> ()
-  end
-
-let _ = at_exit print_memory_stat
```

Comment:
It seems `print_memory_stat` is not called from anywhere?

Reply:
I moved it to `init_execution` (triggered by option `-m`)

Reply:
Umm, I cannot see it in https://github.com/coq/coq/pull/10246/commits/b2a3df4b81bb93302968fc07c6d4fa77e94a5933

Reply:
Indeed, I mixed up lines, probably while rebasing, and the corresponding was in another commit. I fixed it.

Reply:
Thanks!